W-14025615-unifiedflow-botworkflowlibrary (#255)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processautomation-manage-botworkflowlibrary.adoc

= Managing the Bot Workflow Library

> The *Bot Workflow Library* view shows all bot workflows from the RPA Builder Activity Library.

Reviewer: from > in ???

> The *Bot Workflow Library* view shows all bot workflows from the RPA Builder Activity Library.

Reviewer: ```suggestion
The *Bot Workflow Library* view shows all bot workflows in the RPA Builder Activity Library.
```

> You can view and manage the workflows:

Reviewer: ```suggestion
You can do the following to view and manage the workflows:
```

Reviewer: or "You can do the following to view and manage the workflows:"

* Create, edit, and delete tags for the library.
* View an outline of a workflow.
* Edit the description and the tags of a workflow.

== Before You Begin

* Ask an organization administrator in Access Management to assign you the required permissions:
+
[cols="1,1,1"]
|===
|*Action* |*RPA Permission* | *Deprecated RPA Permission*

> |Open the *Process Automation* module

Reviewer: ```suggestion
|Open the *Process Automation* module.
```

|RPA Administrator
|Process Automation Open

> |Open the Bot Workflow Library

Reviewer: ```suggestion
|Open the Bot Workflow Library.
```

|RPA Automations Contributor, RPA Automations Manager
|Activity Library Open

> |Create tags for the library. Add, edit, and remove tags and descriptions from bot workflows

Reviewer: ```suggestion
|Create tags for the library. Add, edit, and remove tags and descriptions from bot workflows.
```

Reviewer: I think that this would be better as bullet list items, which would make this more skimmable.

|RPA Automations Manager
|Activity Library Administration

|===

== Viewing Bot Workflows

Get an overview of all bot workflows.

=== View the List of All Bot Workflows

> Open the *Bot Workflow Library* view in the *Process Automation* module to view the list of all bot workflows and to see their purpose.

Reviewer: ```suggestion
Open the *Bot Workflow Library* view in the *Process Automation* module to view the list of all bot workflows their purpose.
```

> Open the *Bot Workflow Library* view in the *Process Automation* module to view the list of all bot workflows and to see their purpose.

Reviewer: ```suggestion
Open the *Bot Workflow Library* view in the *Process Automation* module to view the list of all bot workflows and see their purpose.
```

=== View an Outline of a Bot Workflow

View the outline of a bot workflow to better understand how it works:

. Open the *Bot Workflow Library* view in the *Process Automation* module.
. Click *Details* image:show-icon.png[eye symbol,1.5%,1.5%] in the panel of the workflow to view.
. Click *Back* to view the list of all bot workflows.

== Managing Bot Workflows

> Change the descriptions and tags of bot workflows.

Reviewer: I would clarify "descriptions" to indicate that it is tag descriptions. Maybe:
Change the tags and tag descriptions of bot workflows.

Author: Both the decriptions and the tags belong to the bot workflows. The tag itself does not have a description, but should be self-explanatory.

Reviewer: OK - thanks for explaining this.

=== Change the Description of a Bot Workflow

Edit the description of a bot workflow tag to render it more precisely:

. Open the *Bot Workflow Library* view in the *Process Automation* module.
. Click *Edit* image:edit-icon.png[pen-to-paper symbol,1.5%,1.5%] in the panel of the workflow to edit.
. In the *Edit <Bot Workflow Name>* window, change the description.
. Click *Update*.

> === Change Tags of a Bot Workflow

Reviewer: ```suggestion
=== Change the Tags of a Bot Workflow
```

Add or remove tags of a bot workflow to define it more precisely:

. Open the *Bot Workflow Library* view in the *Process Automation* module.
. Click *Edit* image:edit-icon.png[pen-to-paper symbol,1.5%,1.5%] in the panel of the workflow to edit.
. In the *Edit <Bot Workflow Name>* window, add or remove tags.
. Click *Update*.

== Managing Bot Workflow Tags

> Create, edit, and delete tags for bot workflows to use in RPA Manager and RPA Builder. Changes affect all tags, regardless of their assignment.

Reviewer: As written, this sentence implies that you can delete tags to use in RPA Manager and RPA Builder. I think you need to split this sentence up. Also, it is not clear if the tags are used in RPA Manager and RPA Builder or the bots are used in these products. Also, why mention RPA Manager and RPA Builder here and not elsewhere?

Author: The implication is correct. Because the [Activity/Bot Workflow] Library is used in both products (unfortunately under different names], changes of the tags and the [activity/bot] workflows affect both products at the same time.
If you delete a tag in RPA Manager, it's gone in RPA Builder as well - and vice versa. If you change the description or the tags of a bot workflow in RPA Builder, the change is shown in RPA Manager as well - and vice versa. The introductory paragraph also mentions RPA Builder. I think it is important that the user knows that his actions might affect more than he can see.

Reviewer: OK - that makes sense. The sentence should be rewritten because as written, it implies that you delete tags to use in RPA Manager and RPA Builder. You delete **from** those products.

=== Create a New Bot Workflow Tag

> Create a new bot workflow tag for categorizing workflows:

Reviewer: ```suggestion
Create a new bot workflow tag to categorize workflows:
```

* Create, edit, and delete tags for the library.
* View an outline of a workflow.
* Edit the description and the tags of a workflow.

== Before You Begin

* Ask an organization administrator in Access Management to assign you the required permissions:
+
[cols="1,1,1"]
|===
|*Action* |*RPA Permission* | *Deprecated RPA Permission*

|Open the *Process Automation* module.
|RPA Administrator
|Process Automation Open

|Open the Bot Workflow Library.
|RPA Automations Contributor, RPA Automations Manager
|Activity Library Open

a|
* Create tags for the library. 
* Add, edit, and remove tags from bot workflows.
* Add and edit bot workflow descriptions.
|RPA Automations Manager
|Activity Library Administration

|===

== Viewing Bot Workflows

Get an overview of all bot workflows.

=== View the List of All Bot Workflows

Open the *Bot Workflow Library* view in the *Process Automation* module to view the list of all bot workflows and see their purpose.

=== View an Outline of a Bot Workflow

View the outline of a bot workflow to better understand how it works:

. Open the *Bot Workflow Library* view in the *Process Automation* module.

> . Click *Details* image:show-icon.png[eye symbol,1.5%,1.5%] in the panel of the workflow to view.

Reviewer 2: For the inline images, I believe there should be parenthesis around them. From the screenshot guide:

Click the Run Now icon (image:icon.png[]).

https://confluence.internal.salesforce.com/display/MTDT/Create+Desktop+Screenshots+for+MuleSoft+Docs
